### PR TITLE
Updated show error on inputs

### PR DESCRIPTION
**Pull request by hasahmad**

### CODE REVIEW

**Comment by xiaohanzhang** [comment]
@hasahmad  what is the use case for showError ?

**Comment by hasahmad** [comment]
It don't want to show it in unfocused view until it is specified.
For example, when filling out the form, its should show on click of input, and when you click create button then show that as well but I don't want to show when not editing or before submitting form.

**Comment by xiaohanzhang** [comment]
In that case, could we only set error on click? 
Basically, only set error when we want show error.

**Comment by hasahmad** [comment]
How about passing in prop for `focusError`
So, when `focusError` is passed in then show onClick?

**Comment by xiaohanzhang** [comment]
I didn't find other usage in Input.tsx, am I missing something ? error is just a boolean in Input component.

**Comment by hasahmad** [comment]
yes, check in the `focus` block in input as well where error is also being used.
I was saying to replace in this block error with another variable so we can have more control on when to show.

**Comment by xiaohanzhang** [comment]
Sorry, I get it now.
I agree focusError is better than showError.

From ui perspective, I am not sure if focusError is necessary. I feels most ui will clear error on focus, instead of show error. But this is more of a UI design issue.

**Comment by hasahmad** [comment]
Yes, true.
@Yakubovich what do you think?

File: src/@commonsku/styles/Sizer.tsx [hunk 1]
```diff
@@ -32,8 +32,7 @@ export type SizerTypes = {
 };
 
 export const SizerCss = css<SizerTypes>`
-    padding: ${(props) => props.padded ? '0.5rem' : 'initial'};
-    margin-left: ${(props) => (props.offset || 0)/12 * 100}%;
+    ${(props) => props.offset && `margin-left: ${(props.offset || 0)/12 * 100}%;`}
```
Comment (xiaohanzhang) [inline src/@commonsku/styles/Sizer.tsx:35]: if we already have "props.offset &&", why we still need "props.offset ||" ?

**Comment by Yakubovich** [comment]
I don't anticipate a use case of showing errors on focus

File: src/@commonsku/styles/Sizer.tsx (hunk 1 above)
Comment (hasahmad) [reply src/@commonsku/styles/Sizer.tsx]: Yes, I've corrected it right now

**Pull request merged**